### PR TITLE
LPS-157180 - Remove metal.js from initial page load

### PR DESCRIPTION
**Pull request by bryceosterhaus**
Lets see how CI likes this

### CODE REVIEW

File: modules/apps/archived/frontend-editor-tinymce-web/src/main/resources/META-INF/resources/tinymce.jsp [hunk 1]
```diff
@@ -143,27 +143,25 @@ name = HtmlUtil.escapeJS(name);
...
-				var openItemSelectorDialog = function (itemSelectorDialog) {
-					itemSelectorDialog.eventName = '<%= name %>selectItem';
-					itemSelectorDialog.singleSelect = true;
-					itemSelectorDialog.url = url;
+				let zIndex;
 
-					var tinymceDialogContainer = document.querySelector(
-						'.tox-tinymce-aux'
-					);
+				var tinymceDialogContainer = document.querySelector(
```
Comment (kresimir-coko) [inline modules/apps/archived/frontend-editor-tinymce-web/src/main/resources/META-INF/resources/tinymce.jsp:148]: `let` and `var` in a JSP. Is that legal?

File: modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/ChangeTrackingIndicator.js [hunk 2]
```diff
@@ -134,16 +134,12 @@ const PublicationsSearchContainer = ({
...
 
-			AUI().use('liferay-portlet-url', () => {
-				const portletURL = Liferay.PortletURL.createURL(
-					saveDisplayPreferenceURL
-				);
+			const portletURL = createPortletURL(saveDisplayPreferenceURL);
 
-				portletURL.setParameter('key', preferencesPrefix + '-' + key);
-				portletURL.setParameter('value', value.toString());
+			portletURL.setParameter('key', preferencesPrefix + '-' + key);
```
Comment (kresimir-coko) [inline modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/ChangeTrackingIndicator.js:139]: I know you haven't changed this line apart from the indentation, but if you end up making any further changes to this file you can use template literals here.

File: modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/ChangeTrackingRenderView.js [hunk 3]
```diff
@@ -790,33 +790,26 @@ export default function ChangeTrackingRenderView({
...
+		const editPortletURL = createPortletURL(editURL);
 
-			editPortletURL.setParameter(
-				'redirect',
-				window.location.pathname + window.location.search
-			);
+		editPortletURL.setParameter(
+			'redirect',
+			window.location.pathname + window.location.search
+		);
```
Comment (kresimir-coko) [inline modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/ChangeTrackingRenderView.js:798]: I think that we can actually set these parameters inside of the initial `createPortletURL` invocation, but might not be something you want to do as part of this PR

File: modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/PortletBase.es.js [hunk 4]
```diff
@@ -24,18 +22,39 @@ function toElementHelper(elementOrSelector) {
...
+
+		this.detached();
+
+		this.disposed();
+	}
+	attached() {}
+	created() {}
+	detached() {}
+	disposed() {}
+	disposeInternal() {}
```
Comment (kresimir-coko) [inline modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/PortletBase.es.js:42]: What's your logic behind this addition/change? I'm not questioning it, I'm just interested as to how you came up to this idea and what makes it good. I don't think I've seen something like this before.

File: modules/apps/archived/frontend-editor-tinymce-web/src/main/resources/META-INF/resources/tinymce.jsp (hunk 1 above)
Comment (bryceosterhaus) [reply modules/apps/archived/frontend-editor-tinymce-web/src/main/resources/META-INF/resources/tinymce.jsp:148]: ah, right. I may have forgotten to run our formatter...

File: modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/PortletBase.es.js (hunk 4 above)
Comment (bryceosterhaus) [reply modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/PortletBase.es.js:42]: you know, I think I may have pushed the wrong branch... this isn't supposed to be here

**Pull request closed without merging**